client/src/i18n/locales/en.json

```diff
@@ -57,5 +57,11 @@
   "FormErrors": {
     "Required": "This field is required.",
     "MaxLength": "Length shouldn't be longer than {max} characters."
+  },
+  "MistakeOptions": {
+    "Label": "Mistake options",
```

Comment:
Thus we decided to remove this text, it is not needed here. But I came up with an idea - you could try to add v-tooltip on a dropdown button (those 3 dots) that has this translation :)